### PR TITLE
Thin OpenSDS installation document and shell script update to support Thin-OpenSDS

### PR DESCRIPTION
**Pull request by Click2Cloud-Gamma**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: docs/Thin-OpenSDS Installation Guide.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+#Thin OpenSDS Installation Guide
```
Comment (leonwanghui) [inline docs/Thin-OpenSDS Installation Guide.md]: Missing a blank space here.

File: docs/Thin-OpenSDS Installation Guide.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+export OS_AUTH_URL=http://$HOST_IP/identity
+export OS_USERNAME=admin
+export OS_PASSWORD=opensds@123
+export OS_TENANT_NAME=admin
+export OS_PROJECT_NAME=admin
+export OS_USER_DOMAIN_ID=default
+```
+
+#### Create a volume using create volume API request
+#####Endpoint
```
Comment (leonwanghui) [inline docs/Thin-OpenSDS Installation Guide.md]: Missing a blank space.

File: docs/Thin-OpenSDS Installation Guide.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+```$xslt
+http://localhost:50040/v1beta/{{ tenant ID }}/block/volumes
+```
+
+#####Request body (update poolName and poolID)
```
Comment (leonwanghui) [inline docs/Thin-OpenSDS Installation Guide.md]: Same here.

File: docs/Thin-OpenSDS Installation Guide.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+osdsctl volume delete <your_volume_id>
+```
+
+
+You can install [CSI Plugin using Helm](https://github.com/opensds/opensds-installer/blob/master/charts/OpenSDS%20Installation%20using%20Helm.md)
+## Uninstall the local cluster
+### OpenSDS(Hotpot)
+It's also nice to uninstall the cluster with one command:
+```
+cd $GOPATH/src/github.com/opensds/opensds && script/devsds/uninstall.sh
```
Comment (leonwanghui) [inline docs/Thin-OpenSDS Installation Guide.md]: It's under install folder.

File: docs/Thin-OpenSDS Installation Guide.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+```
+
+If you want to destroy the cluster, please run the command below instead:
+```
+cd $GOPATH/src/github.com/opensds/opensds && script/devsds/uninstall.sh -purge
```
Comment (leonwanghui) [inline docs/Thin-OpenSDS Installation Guide.md]: Please change script folder to install.

File: docs/Thin-OpenSDS Installation Guide.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+vi install/devsds/local.conf
+```
+
+#### Run osdsapiserver and osdsdock services of OpenSDS using following
+```
+cd $GOPATH/src/github.com/opensds/opensds && install/devsds/install.sh
+```
+If everything goes well, you will get some connection messages at the console output:
+```shell
+Execute commands blow to set up ENVs which are needed by OpenSDS CLI:
```
Comment (kumarashit) [inline docs/Thin-OpenSDS Installation Guide.md]:
```suggestion
Execute commands **below** to set up ENVs which are needed by OpenSDS CLI:
```

File: docs/Thin-OpenSDS Installation Guide.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+------------------------------------------------------------------
+export OPENSDS_AUTH_STRATEGY=keystone
+export OPENSDS_ENDPOINT=http://localhost:50040
+export OS_AUTH_URL=http://10.10.3.150/identity
```
Comment (kumarashit) [inline docs/Thin-OpenSDS Installation Guide.md]:
```suggestion
export OS_AUTH_URL=http://**<host_ip>**/identity
```

File: docs/Thin-OpenSDS Installation Guide.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+export OS_USERNAME=admin
+export OS_PASSWORD=opensds@123
+export OS_TENANT_NAME=admin
+export OS_PROJECT_NAME=admin
+export OS_USER_DOMAIN_ID=default
+------------------------------------------------------------------
+Enjoy it !!
```
Comment (kumarashit) [inline docs/Thin-OpenSDS Installation Guide.md]: Required?

File: docs/Thin-OpenSDS Installation Guide.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+
+##### Request body (update poolName and poolID)
+```
+{
+  "name": "test-1",
+  "description": "volume create test through api",
+  "size": 1,
+  "availabilityZone": "default",
+  "profileId": "",
+  "poolId":"{{ pool ID }]",
```
Comment (kumarashit) [inline docs/Thin-OpenSDS Installation Guide.md]: Is this correct or should it be [{ pool ID }] ?

Comment (leonwanghui) [reply docs/Thin-OpenSDS Installation Guide.md]: Yes, poolId is required in thin-opensds.

**Pull request merged**